rootfs/scheduler/__init__.py

```diff
@@ -622,23 +622,22 @@ def _create_rc(self, name, image, command, **kwargs):
         containers[0]['args'] = args
         loc = locals().copy()
         loc.update(re.match(MATCH, container_fullname).groupdict())
-        mem = kwargs.get('memory', {}).get(loc['c_type'])
-        cpu = kwargs.get('cpu', {}).get(loc['c_type'])
```

Comment:
we can just interchange this with

```
kwargs.get('memory', {}).get(app_type)
```

It's best to set it here rather than in the `if` block so future rewrites have the correct data.